1.8/administration/access-node/tunnel.md

```diff
@@ -114,16 +114,33 @@ DC/OS Tunnel provides you with full access to the DNS, masters, and agents from
 * [SSH access](/docs/1.8/administration/access-node/sshcluster/) (key authentication only).
 * [The OpenVPN client](https://openvpn.net/index.php/open-source/downloads.html) for VPN functionality.
 
-**Note:** Only Linux and OS X are supported currently.
+**Note:** Only Linux and macOS are supported currently.
```

Comment:
It is and isn't required. It's required if you want the SRV record to work, but not if you aren't using SRV records.

It should be clear that SRV records won't work unless you name the port, because then then you won't have `myport` to use in the `curl` example.

Setting up your service is separate from setting up the proxy/vpn.